guides/introduction/Embedded Schemas.md

```diff
@@ -1,89 +1,86 @@
 # Embedded Schemas
 
-Embedded schemas are a feature in Ecto that allows you to define and validate structured data. This data can reside in-memory through structs or, with a migration, be stored in a database field. Embedded schemas work with maps and arrays.
+Embedded schemas allow you to define and validate structured data that is nested within another struct. This data can live exclusively in memory, or can be stored in the database.
```

Comment:
The name here is confusing but the original definition is correct. You don't need to nest an embedded struct within another struct. In fact, if you have in-memory data, you use an embedded schema too!

Reply:
I find the addition of the word exclusively confusing. I understand the meaning you want to imply but I don't think it will be clear for newcomers. For now I recommend reverting this paragraph and keeping the original introduction. :)

Reply:
"spiritually reverted" in my next commit with some other options as well